model/gen-ai/events.yaml

```
@@ -22,6 +42,26 @@ groups:
     brief: >
       This event describes the prompt message specified by the user.
```

```suggestion
      This event describes the user message passed inside the prompt to the GenAI model.
```
Per my previous comment.

---

I want to avoid "passed in the prompt" here too, since we should be able to reuse message events for assistants